app/src/lumen/java/com/cliqz/browser/purchases/PurchaseFragment.kt

```diff
@@ -35,11 +36,14 @@ class PurchaseFragment : DialogFragment(), OnBuyClickListener {
 
     private lateinit var mAdapter: ProductListAdapter
 
-    @set:Inject
+    @Inject
+    lateinit var bus: Bus
+
+    @Inject
     lateinit var purchasesManager: PurchasesManager
 
     @Inject
-    lateinit var bus: Bus
+    lateinit var preferenceManager: PreferenceManager
```

Comment:
the `preferenceManager` is not used in this class

Reply:
Removed it :+1: